flows/organization.md

```diff
@@ -0,0 +1,85 @@
+# Organization flow
+
+Organization is a set of independent users, which could use Classtime.  
+All organization users could be controlled by organization admin.
+
+## Organization admin creation
+
+Only Classtime internally has privileges to create organization admin.  
+Please, contact support@classtime.com to create admin with a school.  
+
+We will provide you with organization **admin access token** and **school id** connected to organization.
+
+## Create teachers
+
+With admin access token you would be allowed to create users within your organization. `Account.createAccount` is a responsible method:
+
+```
+curl -v https://www.classtime.com/service/public/Account/createAccount \
```

Comment:
Note: `service/public` is temporarily not supported. So, remove it for test purposes.